tools/LibTargets.mk

```diff
@@ -194,7 +194,11 @@ libs$(DELIM)libxx$(DELIM)libxx$(LIBEXT): pass1dep
 staging$(DELIM)libxx$(LIBEXT): libs$(DELIM)libxx$(DELIM)libxx$(LIBEXT)
 	$(Q) $(call INSTALL_LIB,$<,$@)
 
+ifeq ($(CONFIG_BUILD_PROTECTED),y)
```

Comment:
Has this been tried CONFIG_BUILD_KERNEL?  It is a similar build in many ways.

Reply:
i haven't tested CONFIG_BUILD_KERNEL.
my understanding is this doesn't matter for CONFIG_BUILD_KERNEL as it doesn't have libapps either in USERLIBS or NUTTXLIBS.